### PR TITLE
FA-Combined Pass & Fail

### PR DESCRIPTION
**Pull request by shenbakavanee-deriv**
Combined both Mobile & Desktop for pass & fail scenario 🙏 

Cards:
1. https://app.clickup.com/t/20696747/QATEST-5159
2. https://app.clickup.com/t/20696747/QATEST-5168

### CODE REVIEW

File: cypress/e2e/wip/Compliance/FA-fail.cy.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+describe('QATEST-5168 MF financial assessment (Appropriateness Test) - Fail scenario but successful account creation', () => {
+  const size = ['small', 'desktop']
+  let country = Cypress.env('countries').ES
+  let nationalIDNum = Cypress.env('nationalIDNum').ES
+  let taxIDNum = Cypress.env('taxIDNum').ES
+  let currency = Cypress.env('accountCurrency').USD
+
+  size.forEach((size) => {
+    it(`Check FA on ${size == 'small' ? 'mobile' : 'desktop'}`, () => {
```
Comment (bilalmajid-deriv) [inline cypress/e2e/wip/Compliance/FA-fail.cy.js:12]:
```suggestion
    it(`should check FA on ${size == 'small' ? 'mobile' : 'desktop'}`, () => {
```

File: cypress/e2e/wip/Compliance/FA-fail.cy.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      cy.findByRole('heading', {
+        name: 'When are you required to pay an initial margin?',
+      }).should('be.visible')
+      cy.findByText('When trading multipliers.').click()
+      cy.findByRole('button', { name: 'Next' }).click()
+
+      if (isMobile) {
+        cy.c_completeFinancialAssessment({ isMobile: true })
+      } else {
+        cy.c_completeFinancialAssessment({})
```
Comment (bilalmajid-deriv) [inline cypress/e2e/wip/Compliance/FA-fail.cy.js:96]:
```suggestion
        cy.c_completeFinancialAssessment()
```
You will not need to send and empty object here

File: cypress/e2e/wip/Compliance/FA-pass.cy.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+describe('QATEST-5159 MF financial assessment (Appropriateness Test) - Pass scenario and successful account creation', () => {
+  const size = ['small', 'desktop']
+  let country = Cypress.env('countries').ES
+  let nationalIDNum = Cypress.env('nationalIDNum').ES
+  let taxIDNum = Cypress.env('taxIDNum').ES
+  let currency = Cypress.env('accountCurrency').USD
+
+  size.forEach((size) => {
+    it(`Check FA on ${size == 'small' ? 'mobile' : 'desktop'}`, () => {
```
Comment (bilalmajid-deriv) [inline cypress/e2e/wip/Compliance/FA-pass.cy.js:12]:
```suggestion
    it(`should check FA on ${size == 'small' ? 'mobile' : 'desktop'}`, () => {
```

File: cypress/e2e/wip/Compliance/FA-pass.cy.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+      cy.findByRole('heading', {
+        name: 'When are you required to pay an initial margin?',
+      }).should('be.visible')
+      cy.findByText('When opening a leveraged CFD trade.').click()
+      cy.findByRole('button', { name: 'Next' }).click()
+
+      if (isMobile) {
+        cy.c_completeFinancialAssessment({ isMobile: true })
+      } else {
+        cy.c_completeFinancialAssessment({})
```
Comment (bilalmajid-deriv) [inline cypress/e2e/wip/Compliance/FA-pass.cy.js:96]:
```suggestion
        cy.c_completeFinancialAssessment()
```

**Review by bilalmajid-deriv — COMMENTED** [review]
Minor changes

**Review by abuzar-deriv — APPROVED** [review]
LGTM apart from the pending comments.

**Review by bilalmajid-deriv — APPROVED** [review]

**Review by maliheh-deriv — APPROVED** [review]

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**